frontend/comfywatchman-dashboard/README.md

```
@@ -0,0 +1,10 @@
+
+  # ComfyUI Model Management Dashboard
```

markdownlint-cli2 (0.22.1)

[warning] Headings must start at the beginning of the line

(MD023, heading-start-left)